infra/k8s/install.sh

```diff
@@ -66,19 +54,70 @@ while [ "$READY_NODES" -ne $(($WORKER_NODES + 1)) ]; do READY_NODES=$(kubectl ge
 echo "Cluster nodes are Ready"
 echo
 
-echo "Add secret for S3 bucket"
-echo
-kubectl create secret generic assets-s3-bucket --from-literal=access-key="$ASSETS_ACCESS_KEY" \
-                                               --from-literal=secret-key="$ASSETS_SECRET_KEY" \
-                                               --from-literal=s3-endpoint="$ASSETS_S3_ENDPOINT" \
-                                               --from-literal=bucket-name="$ASSETS_BUCKET_NAME"
+echo "Install EFS..."
 
+vpcId=`aws ec2 describe-vpcs --filters Name=tag:Name,Values=$NAME | jq ".Vpcs | .[] | .VpcId" | tr -d "\""`
+
+if [[ -z ${vpcId} ]]; then
+  echo "Couldn't detect AWS VPC created by `kops`"
+  exit 1
+fi
+
+echo "Detected VPC: $vpcId"
+
+securityGroupId=`aws ec2 describe-security-groups | jq ".SecurityGroups | .[] | select(.GroupName==\"nodes.$NAME\") | .GroupId" | tr -d '"'`
+
+if [[ -z ${securityGroupId} ]]; then
+  echo "Couldn't detect AWS Security Group created by `kops`"
+  exit 1
+fi
+
+echo "Detected Security Group ID: $securityGroupId"
+
+subnetId=`aws ec2 describe-subnets | jq ".Subnets | .[] | select (.VpcId == \"$vpcId\") | .SubnetId" | tr -d '"'`
+
+if [[ -z ${subnetId} ]]; then
+  echo "Couldn't detect AWS Subnet created by `kops`"
+  exit 1
+fi
+
+echo "Detected Subnet ID: $subnetId"
+
+pushd efs-terraform
+
+# extract s3 bucket from kops state store
+S3_BUCKET="${KOPS_STATE_STORE:5:100}"
```

Comment:
KOPS_STATE_STORE has the format of `s3://bucket-name`, whereas here we need `bucket-name`.